### PR TITLE
Improve the masterclasses page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Updated the site's navigation
- Pulled the upcoming sessions from the sheet
- Updated the masterclasses template to include upcoming list and information about signing up

## QA
- You will likely need creds to pop in your `.local.env` - **Ping me for them**
- Open /masterclasses
- Check it looks ok
- Check the information presented matches the [data source](https://docs.google.com/spreadsheets/d/1fFumFWIM3oHwLr9pcBlaANcadAU0bNJxbkfGKwC_1pg/edit#gid=0)

### CODE REVIEW

File: templates/shared/_card--masterclass.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<div class="p-card u-no-padding col-4">
+<div class="p-card u-no-padding col-3">
   {% if session.Link %}
   <a href="{{ session.Recording }}" aria-hidden="true" tabindex="-1">
     <img class="p-card__image" alt="{{ session.Topic }} video" src="https://lh3.google.com/u/0/d/{{ session.Link }}=k">
   </a>
   <div class="p-card__inner">
-    <h3><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted">Duration: {{ session.Duration }}</p>
+    <h5 class="u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h5>
```
Comment (sowasred2012) [inline templates/shared/_card--masterclass.html:7]: There's a jump from heading level 2 to 5 here
```suggestion
    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
```

File: templates/shared/_card--masterclass.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h3><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
   </div>
   {% else %}
   <div class="p-card__inner">
-    <h3>{{ session.Topic }}</h3>
+    <h5 class="u-no-margin--bottom">{{ session.Topic }}</h5>
```
Comment (sowasred2012) [inline templates/shared/_card--masterclass.html:17]:
```suggestion
    <h3 class="p-heading--5 u-no-margin--bottom">{{ session.Topic }}</h3>
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
One heading level tweak - also, I'm not sure why but none of the thumbnails are rendering for me? It's not specific to this PR, I also saw it on #108

image

**Comment by anthonydillon** [comment]
Thanks, @sowasred2012 I have addressed your comments and added a default image if the image from google fails. I believe your Google session might have run out or you are missing some permissions to the video directory. If that is the case I need to open the directory to everyone.

**Review by sowasred2012 — APPROVED** [review]
Nice! Looks good to me - I'm pretty sure my Google session is still active, I don't have issues with gmail, calendar, drive etc, so maybe it is permissions? The default image works though.

**Pull request merged**